shared/actions/gregor.js

```diff
@@ -52,6 +57,14 @@ function registerGregorListeners () {
       dispatch(pushState(state, reason))
       response && response.result()
     })
+
+    engine().setIncomingHandler('keybase.1.gregorUI.pushOutOfBandMessages', ({oobm}, response) => {
+      if (oobm) {
```

Comment:
wouldn't it be better to do the length check after the filter so we don't push the action if there's zero? or do we actually need to push to clear out some state and therefore we shouldn't have the if at all?